src/NuGet.Clients/NuGet.VisualStudio.Client/source.extension.vs15.insertable.vsixmanifest

```diff
@@ -23,6 +23,7 @@
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.Console" Path="|NuGet.Console|" />
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.VisualStudio.Implementation" Path="|NuGet.VisualStudio.Implementation|" />
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.SolutionRestoreManager" Path="|NuGet.SolutionRestoreManager|" />
+        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGet.VisualStudio.Common" Path="|NuGet.VisualStudio.Common|" />
```

Comment:
Ahh we had to add it to MEFComponent list. But would it have any adverse effect on VS initialization or load for an additional MEF? As I've mentioned multiple times, this change doesn't make any difference on ground level since `NuGet.SolutionRestoreManager` is always the first NuGet package being loaded inside VS.

Reply:
I know. I will run RPS before checking this in.

Reply:
RPS result for this branch is [here](http://ddweb/Dashboard/VSBuild/detail/val/27916.9000.nuget-client.180716.105845).